src/containers/AdminCampaignStats.jsx

```diff
@@ -424,9 +424,10 @@ class AdminCampaignStats extends React.Component {
           message={
             <span>
               Export started -
-              {this.props.organizationData &&
-                this.props.organizationData.emailEnabled &&
-                " we'll e-mail you when it's done. "}
+              {(this.props.organizationData &&
+                this.props.organizationData.emailEnabled) ?
+                " we'll e-mail you when it's done. " :
+                " check your Spoke directory. "}
```

Comment:
I just tried this locally, if we have AWS S3 set up, even if email isn't enabled, the data will save to S3, not in the spoke directory

Reply:
disregard my previous comment, the last change to this check fixed the issue I was seeing